src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -565,8 +565,13 @@ std::shared_ptr<ov::ICompiledModel> Plugin::import_model(std::istream& model_str
         decript_from_string = true;
     }
 
+    std::shared_ptr<ov::AlignedBuffer> model_buffer;
```

Comment:
```suggestion
    std::shared_ptr<ov::AlignedBuffer> model_buffer = nullptr;
```

Reply:
shared_ptr default ctr assigns it to nullptr